docs/source/new-docs/contribute/adding-functionality/contributors.rst

```diff
@@ -0,0 +1,47 @@
+.. dropdown:: Distribution Statement
+
+ | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+ | # # #
+ | # # # Author:
+ | # # # Naval Research Laboratory, Marine Meteorology Division
+ | # # #
+ | # # # This program is free software: you can redistribute it and/or modify it under
+ | # # # the terms of the NRLMMD License included with this program. This program is
+ | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+ | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+ | # # # for more details. If you did not receive the license, for more information see:
+ | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+##################
+Contributors Guide
+##################
+
+The GeoIPS Team encourages active participation by the user and developer community,
+so we welcome any and all feedback on and updates to the code base.
+
+Each step in this process can be completed by one or more individuals
+(we encourage collaboration!), so feel free to submit Issues even if
+you have no intention of resolving it yourself. And feel free to
+submit a Pull Request even if all tests are not complete - someone
+else may have time to finalize for approval even if you are unable.
+
+Contribution Process
+====================
+
+Contributions generally follow the process of:
+
+1. Submit a GitHub Issue for a bug fix or feature request. :ref:`create_issue`
```

Comment:
This unfortunately is veering from how to contribute and into how to use git/github.

Reply:
This is what was in the documentation previously. I don't see the harm in demonstrating our process of how to contribute to GeoIPS. Would you agree on that? Git is the only way you can actively contribute code to GeoIPS.